### PR TITLE
Add Python 3 support and test on Travis CI

### PR DESCRIPTION
**Pull request by hugovk**
Please enable Travis CI for this repo at https://travis-ci.org/profile/

See .travis.yml for what it does: some static analysis and then does a test run of `python generate.py`.

Here's an example build: https://travis-ci.org/hugovk/pythonwheels/builds/283534796

### CODE REVIEW

**Comment by meshy** [comment]
@hugovk thank you for this!

I hope you don't mind if I ask for a few alterations:

- It's a great idea to move this to python 3. I don't think there's much need to support python 2 at the same time, however. Would you mind removing it?
- Instead of `pyflakes` and `pep8` separately, could you please use `flake8`?

Once again, this is much appreciated. Thank you!

**Comment by hugovk** [comment]
Updated!

File: utils.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
+from __future__ import print_function
```
Comment (ghickman) [inline utils.py]: IIRC this shouldn't be necessary under Python 3?

File: utils.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 import datetime
 import json
-import xmlrpclib
-
 import pytz
 import requests
+import xmlrpc.client
```
Comment (ghickman) [inline utils.py]: This is in the stdlib on Python 3, keep the stdlib imports grouped please.

**Comment by hugovk** [comment]
@ghickman Thanks, both updated.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+script:
+ # Static analysis
+ - flake8 --statistics --count .
+
+ # Test run
+ - python generate.py
+
+matrix:
+  fast_finish: true
```
Comment (meshy) [inline .travis.yml]: One last thing, could I please get you to add:

```
notifications:
  email: false
```

**Comment by hugovk** [comment]
@meshy Updated!

**Comment by meshy** [comment]
You're a star! Thank you!

**Pull request merged**